file-mod/cmd/replace.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/chef/ci-studio-common/lib"
+
+	"github.com/spf13/cobra"
+)
+
+var replaceCmd = &cobra.Command{
+	Use:   "find-and-replace REGEX_STR STRING FILE",
+	Short: "Replace REGEX_STR with STRING in FILE. Supports multiline replace.",
+	Args:  cobra.MinimumNArgs(3),
```

Comment:
[nit] I believe there's an _exact_ variant, too. Not sure if you care about not breaking backcompat here.